modules/srvrlss-using-cluster-logging-find-knative-serving-logs.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
+
+[id=using-cluster-logging-to-find-Knative-Serving-logs_{context}"]
+= Using Cluster Logging to find logs for Knative Serving components
+
+.Prerequisites
+
+* xref:../efk-logging-about_{context}[Cluster Logging components] are installed.
+
+.Procedure
+
+. To open the Kibana UI (the visualization tool for Elasticsearch), use the following command to get the Kibana route:
```

Comment:
I would suggest removing the parentheses.

Reply:
Also idk if we're supposed to use `:` at the end of sentences?